pybloqs/block/table_formatters.py

```diff
@@ -53,12 +53,18 @@ class TableFormatter(object):
         Provides CSS styles to all <th> and <td> HTML tags.
     """
 
-    def __init__(self, rows=None, columns=None, apply_to_header_and_index=True):
-        """Initialise formatter and specify which rows and columns it is applied to. Default None applies to all."""
-        self.rows = rows
-        self.columns = columns
-        self.apply_to_header_and_index = apply_to_header_and_index
-        return
+    def __init__(self, rows=None, columns=None, apply_to_header_and_index=True, apply_to_header=False, apply_to_index=False):
+            """Initialise formatter and specify which rows and columns it is applied to. Default None applies to all."""
+            self.rows = rows
+            self.columns = columns
+            self.apply_to_header_and_index = apply_to_header_and_index
```

Comment:
Do we need to keep this state now? Can we instead just use self.apply_to_header and self.apply_to_index.